css/style.css

```diff
@@ -0,0 +1,75 @@
+
+.header {
+    display: flex;
```

Comment:
As you can see, you have a repeated property display: flex. You can create a class, for example, .flex, and use it each tome when you need it in html, which reduces duplication of css. Think of other classes for properties, which can be then reused many times. Put it in the beginning of the file for convenience.

Reply:
fixed